### DISCUSSION TITLE
Is the chart supporting airflow version 2.3.0?

### DISCUSSION THREAD

**Discussion by HazalCiplak** [airflow-helm/charts#591] [Questions & Answers] [answered]
We are using this chart in our cluster and recently upgraded airflow to version 2.3.0. But we are not sure if the chart is supporting this version of airflow or when can it start to support?
Thanks.

**Comment by thesuperzapper** [comment, marked as answer]
Yes, chart version 8.6.1 will support Airflow 2.3.0, see more information in the tracking issue https://github.com/airflow-helm/charts/issues/572.

_PS: I am very close to cutting 8.6.1_

**Reply by HazalCiplak** [reply]
Thank you!
